grafana/resource_cloud_stack.go

```diff
@@ -307,6 +307,19 @@ func FlattenStack(d *schema.ResourceData, stack gapi.Stack) {
 	d.Set("alertmanager_status", stack.AmInstanceStatus)
 }
 
+// Append path to baseurl
+func appendPath(baseUrl, path string) string {
+	bu, err := url.Parse(baseUrl)
+	if err != nil {
+		diag.FromErr(err)
```

Comment:
You could use `tflog` to print out an error in that case: https://www.terraform.io/plugin/log/writing

You should also return an empty string, otherwise, line 316 will be a nil reference and panic

Reply:
Bear with me as I'm not much of a Golang nor Terraform developer. I think I understand your comments but not sure how to make it work. Do I need to pass in the context ctx from ReadStack via FlattenStack to make tflog work in appendPath?

Reply:
Oh, you do need the ctx. Might as well return an error then. `FlattenStack` should have an `error` return value and appendPath should have `(string, error)`